src/sections/Resources/Resources-grid/options.js

```diff
@@ -4,17 +4,16 @@ export const options =  [
     subdata: [
       { id: 0, value: "Article", label: "Article" },
       { id: 1, value: "Blog", label: "Blog" },
-      { id: 2, value: "Case Study", label: "Case Study" },
-      { id: 3, value: "Demo", label: "Demo" },
-      { id: 4, value: "FAQ", label: "FAQ" },
-      { id: 5, value: "Guide", label: "Guide" },
-      { id: 6, value: "Interview", label: "Interview" },
-      { id: 7, value: "News", label: "News" },
-      { id: 8, value: "Podcast", label: "Podcast" },
-      { id: 9, value: "Presentation", label: "Presentation" },
-      { id: 10, value: "Recorded Webinar", label: "Recorded Webinar" },
-      { id: 11, value: "Tutorial", label: "Tutorial" },
-      { id: 12, value: "White Paper", label: "White Paper" }]
+      { id: 2, value: "News", label: "News" },
```

Comment:
Any particular reason why we don't have this in alphabetical order anymore?

Reply:
@warunicorn19 I wanted to push the "(0)"s to the end. We can switch it back later once we have each of those collections displayed.